misk/web/@misk/components/src/builders/indexBuilder.tsx

```diff
@@ -0,0 +1,68 @@
+///<reference types="webpack-env" />
+import { IWindow } from "@misk/common"
+import { connectRouter, routerMiddleware } from "connected-react-router"
+import { createBrowserHistory, History } from "history"
+import * as React from "react"
+import * as ReactDOM from "react-dom"
+import { AppContainer } from "react-hot-loader"
+import { Provider } from "react-redux"
+import { AnyAction, applyMiddleware, compose, createStore, Reducer } from "redux"
+import createSagaMiddleware from "redux-saga"
+import { AllEffect } from "redux-saga/effects"
+
+export const indexBuilder = (
```

Comment:
Is it typical to name something like this a builder?

Reply:
I've seen some functions like `createSagaMiddleware` that use `create` as prefix so could move it to that.